index.html

```diff
@@ -119,12 +119,14 @@
         </section>
 
         <section>
-          <img src="http://placehold.it/250x250">
```

Comment:
I tried but it squishes my face
On Mar 7, 2016 10:34 AM, "Tanya Powell" <notifications@github.com> wrote:

> In index.html
> <https://github.com/womenhackfornonprofits/womens-day-panel/pull/11#discussion_r55185780>
> :
>
> > @@ -119,12 +119,14 @@
> >          </section>
> >
> >          <section>
> > -          <img src="http://placehold.it/250x250">
>
> could you add id="bio-pic" to your <img> tag pretty please [image:
> :dancer:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/womenhackfornonprofits/womens-day-panel/pull/11/files#r55185780>
> .
>